.gitignore

```diff
@@ -4,4 +4,5 @@ bin/configlet
 bin/configlet.exe
 output.json
 user_output.json
-user_output.md
\ No newline at end of file
+user_output.md
+.vscode
```

Comment:
I didn't think we should be adding all the various editor and IDE specific files here

Reply:
I should google if there is a local gitignore like thing

Reply:
IMO, your editor's files should be ignored in `~/.config/git/ignore`